spring-integration-jmx/src/test/java/org/springframework/integration/jmx/UpdateMappingsTests.java

```diff
@@ -95,7 +95,7 @@ public void testJmx() throws Exception {
 		MessagingTemplate messagingTemplate = new MessagingTemplate();
 		messagingTemplate.setReceiveTimeout(1000);
 		Set<ObjectName> names = this.server.queryNames(ObjectName
-				.getInstance("update.mapping.domain:type=HeaderValueRouter,name=router"),
+				.getInstance("update.mapping.domain:type=MessageHandler,name=router,bean=endpoint"),
```

Comment:
So, don't we have already the change to use the real class name for the `type` ?
Seems for it is enough useful from JMX tree.